src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -778,6 +778,18 @@ public void scrollToPosition(String selector) {
     jsActions.scrollToSpecificElement(driver.findElement(By.cssSelector(selector)));
     PageObjectLogging.log("scrollToSelector", "Scroll to the web selector " + selector, true);
   }
+  // This scroll has been created becouse ad is not displayed if we scroll quickly to the Footer ADEN-4359
+  public void scrollToBottomLeaderboard(){
+    scrollToFooter();
+    scrollTo(presentLeaderboard);
+    scrollToFooter();
+  }
+
+  public void clickOnAdImage(String slotName){
+    wait.forElementVisible(By.cssSelector(AdsContent.getSlotSelector(slotName)));
```

Comment:
This test works only because when selenium driver is clicking on the `TOP_LEADERBOARD` the point where it clicks is not a video trigger. Can we use a selector that represents the "click through" background of VUAP?

If not, let's leave it as it is. It's highly probable that we will have only one test campaign